### ISSUE TITLE
How to automatically put skin in <>

### ISSUE THREAD

**Issue by matthewaptaylor** [CigarProject/Cigar#507]
So I know that to use a skin, you have to put it's name in brackets. How can I get cigar to use javascript to automatically do that? Or how can I get the skin name to not show up at all, and be stored in a hidden text box, so when I press play Cigar automatically merges the hidden skin textfield with the name field?

**Comment by Y4ssine** [comment]
i need help for this too :/ :-1:

**Comment by Luka967** [comment]
Easiest way is to add this line [here](https://github.com/CigarProject/Cigar/blob/master/www/assets/js/main_out.js#L36):
```js
document.getElementById("nick").innerHTML = "<skin>";

**Comment by Y4ssine** [comment]
that seems to not work can u check my codes ? http://agarcoinscollecter.tk/

2017-01-30 3:37 GMT-08:00 Luka967 <notifications@github.com>:

**Comment by HAlexTM** [comment]
@Y4ssine you removed the repo link, we won't give you support

**Comment by matthewaptaylor** [comment]
Managed to do it: I changed include/gallery.php to input the skin name to the textbox ID "skin" instead of "nick", and then changed the play button to, instead get it's name from text ID "nick", to get it from ID "merge". I created "skin" and "merge", and used javascript so that when I press play, instead of getting nothing from ID "merge", but it combines a left-angle bracket, the input from "skin", a right angle-bracket, the "nick", and then puts it into textbox "merge". Then, it uses the id "merge" textbox, which looks something like "<doge>MyName", as the player name. Easy! (Note sarcasm).
